editor.planx.uk/package.json

```diff
@@ -16,6 +16,7 @@
     "@mui/utils": "^5.15.11",
     "@opensystemslab/map": "1.0.0-alpha.4",
     "@opensystemslab/planx-core": "git+https://github.com/theopensystemslab/planx-core#ccf9ac3",
+    "@tanstack/react-query": "^5.61.3",
```

Comment:
As discussed in dev call, have got rid of react-query for now (and this work in the previous state saved off in another branch [jh/download-application-with-tanstack-react-query](https://github.com/theopensystemslab/planx-new/tree/jh/download-application-with-tanstack-react-query)) until we have figured out if it will work for us on a longer term basis!